### PR TITLE
Integrate Mayhem

### PR DESCRIPTION
**Pull request by rnshah9**

### CODE REVIEW

File: config/goaccess.conf [hunk 1]
```diff
@@ -10,7 +10,7 @@
 # The following time format works with any of the
```
Comment (xansec) [inline config/goaccess.conf]: Is this change needed for fuzzing?

File: mayhem/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,29 @@
+FROM alpine:3.13 as builder
```
Comment (xansec) [inline mayhem/Dockerfile:1]: Perhaps you could add your own goaccess for fuzzing?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: config/goaccess.conf (hunk 1 above)
Comment (rnshah9) [reply config/goaccess.conf]: Yep. Moved it to the Dockerfile, though.

File: mayhem/Dockerfile (hunk 2 above)
Comment (rnshah9) [reply mayhem/Dockerfile:1]: Sorry, what do you mean by my "own goaccess"?

File: mayhem/Mayhemfile_goaccess [hunk 3]
```diff
@@ -0,0 +1,5 @@
+project: goaccess
```
Comment (xansec) [inline mayhem/Mayhemfile_goaccess:1]: You may want to also use `/dev/null` as your output here

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Review by xansec — APPROVED** [review]
LGTM!
